data/sql/updates/pending_db_world/rev_1611179465393657785.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611179465393657785');
+UPDATE `quest_template_addon` SET `PrevQuestID` = -944 WHERE `ID`  IN (960, 961);
```

Comment:
What dows negative prev quest do? Never seen it before

Reply:
PrevQuestID

    if value > 0: Contains the previous quest id, that must be completed before this quest can be started.
    If value < 0: Contains the parent quest id, that must be active before this quest can be started.

Reply:
me neither, I've just saw the comments that wizzymore posted in the wiki and used it.
I still ahve to make some local test, but I think that it's correct